core/src/main/java/org/openstack4j/model/compute/actions/RebuildOptions.java

```diff
@@ -40,6 +45,19 @@ public RebuildOptions name(String name) {
         add(Option.NAME, name);
         return this;
     }
+    
+    public RebuildOptions userData(String userData) {
+        add(Option.USER_DATA, userData);
+        return this;
+    }
+    
+    public void personality(List<Personality> personality) {
```

Comment:
Please make those "setter" methods return `RebuildOptions` to keep the [interface fluent](https://en.wikipedia.org/wiki/Fluent_interface).

Reply:
It was mistake. I added a return type.